easypost/webhook.py

```diff
@@ -29,28 +30,33 @@ def update(self, **params) -> "Webhook":
         return self
 
     @staticmethod
-    def validate_webhook(webhook_body: Dict[str, Any], headers: Dict[str, Any], webhook_secret: str) -> bool:
+    def validate_webhook(event_body: bytes, headers: Dict[str, Any], webhook_secret: str) -> Dict[str, Any]:
         """Validate a webhook originated from EasyPost by comparing the HMAC header to a shared secret.
         This function will return True if the webhook originated from EasyPost and False if it did not.
         """
-        secret_validated = False
+        webhook_body = {}
```

Comment:
this is dead code, right?

Reply:
Indeed, replaced by the else statement. Removed.